File: 2274-keep-multiplying-found-values-by-two/2274-keep-multiplying-found-values-by-two.py
class Solution(object):
    def findFinalValue(self, nums, original):
        """
        :type nums: List[int]
        :type original: int
        :rtype: int
        """
        n=len(nums)
        nums.sort()
        i=0
        while i<n:
            if nums[i]==original:
                original=original*2
            i+=1
        return original

        

File: 2274-keep-multiplying-found-values-by-two/2274-keep-multiplying-found-values-by-two.cpp
class Solution {
public:
    int findFinalValue(vector<int>& nums, int original) {
   int i=0;
   int o=original;
   int n=nums.size();
   sort(nums.begin(),nums.end());
;   while(i<n)
   {
       if(nums[i]==o)
       {
           o=o*2;
       }
       i++;
   }
   return o;

       
    }
};